readme.md

```diff
@@ -0,0 +1,22 @@
+# new feature-development branch has been created
+
+* Creating branch is very mandatory for collabrating remote software developers. 
+This 'Hello Microverse' project is the first day project for Microverse students
+Therefore, this project stated about the how to use the necessary tools and packges in project developemt.
+## The main tools and packges that should be included in this project and in the next are:
+* Linter 
+* workfolw
+## Linter
+* Linter is a a package that used to check code process that and looks for errors in the source code, at end flags potential bugs.
+Resource for [Stylelint](https://stylelint.io/) and [10 CSS and JavaScript Linting Tools for Code Optimization](https://www.hongkiat.com/blog/code-optimization-css-js-linting-tools/)
+## Workflow
+* workflow describes the general steps for developing the project. according tho Microverse moudle the main branch should always be stable code that actually works and can be safely used. Any changes required in the application should be introduced in feature branches and approved before merging them to the main branch.
+[workflows](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
+# Prerequisites
+## In order to run this project you need:
+* Code editor such as vs code, subline, notpade++, dreamwiver or othe modern code editor are needed. 
+
+
+# Autor
+Brhanu Hailu
+
```

Comment:
- [ ] Great job adding some documentation in the README.md as suggested earlier 👍 However, I think it would be better if you follow the [readme template](https://github.com/microverseinc/readme-template) for crafting a professional and high quality README.md ✌📝 
Please note that a professional and well-written README can help users understand the purpose and functionality of the project and it also convey a sense of professionalism and competence, which can help attract users and contributors to the project.
- [ ]  As an example, you can visit 👉 [this README](https://github.com/KanzaTahreem/OOP-School-Library) but please make sure to customize the content according to you project. I know you can do that 💪